apps/web/app/api/v1/client/[environmentId]/app/sync/[userId]/route.ts

```diff
@@ -189,11 +178,14 @@ export async function GET(
       }),
     };
 
+    const language = await getAttribute("language", person.id);
+
     // return state
     const state: TJsAppStateSync = {
-      person: personData,
+      person,
```

Comment:
I don't think we need to return the person here since it doesn't have helpful information for the SDK since we only need the userId to make calls from the SDK. I would remove it (or remove it for newer version of Formbricks.js > 1.65) and store the `userId` in the state of formbricks/js/app